docs/installation/get-started.md

```diff
@@ -0,0 +1,90 @@
+# Get started
+
+Get one GOV.UK Frontend component working in your application, so you can test everything works before you add more components or styles.
+
+You will need to do all the following steps to get your component working.
+
+1. Install GOV.UK Frontend.
+2. Add the HTML for a component to your application.
+3. Get the CSS working.
+4. Get the font and images working.
+5. Get the JavaScript working.
+
+## 1. Install
+
+[Install GOV.UK Frontend using npm](/docs/installation/installing-with-npm.md).
+
+If you've installed from `dist`, get started with [a basic page](/docs/installation/installing-from-dist.md#2-include-resources) instead.
+
+## 2. Add the HTML for a component to your application
+
+Go to the [example accordion component](https://design-system.service.gov.uk/components/accordion/#accordion-example) on the GOV.UK Design System website, then copy the HTML.
+
+Paste the HTML into a page or template in your application.
+
+## 3. Get the CSS working
+
+1. Add the following to the main Sass file in your project, so your Sass compiler adds all of GOV.UK Frontend's styles to your CSS file.
+
+    ```Scss
+    @import "node_modules/govuk-frontend/govuk/all";
+    ```
+
+2. Add your CSS file to your page layout if you need to. For example:
+
+    ```html
+      <head>
+        ...
+        <link rel="stylesheet" href="<YOUR-CSS-FILE>.css">
```

Comment:
Cross referencing a conversation from another PR: https://github.com/alphagov/govuk-frontend-docs/pull/2#discussion_r380771988 We should mention here that if they use the legacy repos, GOV.UK Frontend stylesheet should be included after the legacy files in their page template.

Reply:
I'm not sure we should mention this in Get started, as this doc is meant as a very basic 'get started from scratch' guide. If we started to talk about using legacy repos, we'd also need to mention compatibility mode etc.

Instead, I think it would work if we add this in both:

a) the migration docs (for users migrating)
b) the compatibility mode docs (for users running FE alongside legacy longer-term)

What do you think?

Reply:
Yes adding it to the migration docs would make sense thanks @m-green 👍